examples/local.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Added a local config for debugging without patroni.

Reply:
awesome!